packages/web3-utils/src/index.js

```diff
@@ -292,13 +292,10 @@ var toWei = function(number, unit) {
  * @return {String}
  */
 var toChecksumAddress = function (address) {
-    if (typeof address === 'undefined') return '';
-
-    if(!/^(0x)?[0-9a-f]{40}$/i.test(address))
+    if(typeof address !== 'string' || !/^(0x)?[0-9a-f]{40}$/i.test(address))
```

Comment:
There is a function called `isAddress` I woudl prefeer using it instead of repeating the same regex here.